instructors.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'People'
+
+class Instructors < People
```

Comment:
Use singular names, not plural (unless your class is actually a collection of things).

i.e. `Instructor` and `Person`.